.github/workflows/applications.yml

```diff
@@ -168,7 +168,7 @@ jobs:
       run: cabal build
     - name: Test
       shell: bash
-      run: cabal test +RTS -N
+      run: cabal run tests
```

Comment:
alternatively, since we have some time to look at this deeply, perhaps we should take a look at https://hackage.haskell.org/package/hspec-core-2.11.0.1/docs/Test-Hspec-Core-Spec.html#v:sequential and https://hackage.haskell.org/package/hspec-core-2.11.0.1/docs/Test-Hspec-Core-Spec.html#v:parallel when running these tests in the actual `PactTests.hs` file, rather than relying on CI to do "the right thing". I'll take a stab at this today and if nothing turns up, let's merge as is, but ideally we'd have these be reproducible at the local level via `cabal test`.

Reply:
How did that analysis turn out, Emily?

Reply:
Hard to say - i can't get anything to trigger locally.